### PR TITLE
Fix user keybindings template to use array rather than object

### PR DESCRIPTION
**Pull request by rchl**
Also remove reference to LSP as it's a global keybindings file, not
specific to LSP.

### CODE REVIEW

File: Commands/Default.sublime-commands [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "args": {
             "base_file": "${packages}/LSP/Keymaps/Default (${platform}).sublime-keymap",
             "user_file": "${packages}/User/Default ($platform).sublime-keymap",
-            "default": "// Settings in here override those in \"LSP/Keymaps/Default (${platform}).sublime-keymap\",\n\n{\n\t$0\n}\n"}
+            "default": "[\n\t$0\n]\n"}
```
Comment (rwols) [inline Commands/Default.sublime-commands:15]: This seems to have an extraneous } at the end?

Comment (rchl) [reply Commands/Default.sublime-commands:15]: Not extraneous but in the wrong place.

File: Commands/Default.sublime-commands [hunk 2]
```diff
@@ -4,15 +4,16 @@
         "command": "edit_settings",
         "args": {
           "base_file": "${packages}/LSP/LSP.sublime-settings",
-          "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\",\n\n{\n\t$0\n}\n"}
+          "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\"\n\n{\n\t$0\n}\n"}
```
Comment (rwols) [inline Commands/Default.sublime-commands:7]: Can you also remove the extra newline?

`\n\n{\n\t$0\n}\n"}` -> `\n{\n\t$0\n}\n"}`

File: Menus/Main.sublime-menu [hunk 3]
```diff
@@ -16,15 +16,16 @@
                 "command": "edit_settings",
                 "args": {
                   "base_file": "${packages}/LSP/LSP.sublime-settings",
-                  "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\",\n\n{\n\t$0\n}\n"
+                  "default": "// Settings in here override those in \"LSP/LSP.sublime-settings\"\n\n{\n\t$0\n}\n"
```
Comment (rwols) [inline Menus/Main.sublime-menu:19]: See above.

**Pull request merged**